.tree-sitter-lint.yml

```diff
@@ -597,6 +600,10 @@ rules:
           module: squalid
           kind: trait_method
           trait: OptionExtDeref
+        # sorted_by_key:
```

Comment:
Looks like itertools also exposes a `.sorted()`/`.sorted_by_key()` method (which may be strictly preferable? Not sure)